{
    "repository": "yamonc/New-Pytorch-Chinese",
    "clone_url": "https://github.com/yamonc/New-Pytorch-Chinese.git",
    "timestamp": "2023-05-29T16:28:30.923039Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Python application/build] \ud83d\ude80  Start image=crawlergpt:latest\n[Python application/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Python application/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Python application/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Python application/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/yamonc-New-Pytorch-Chinese] user=0 workdir=\n[Python application/build]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v2\n[Python application/build] Unable to checkout 75f3110429a8c05be0e1bf360334e4cced2b63fa: invalid checksum\n[Python application/build] \ud83c\udfc1  Job succeeded\n",
    "actions_stderr": "Error: invalid checksum\n"
}